main/job-stats.js

```diff
@@ -24,6 +29,19 @@ class JobStats {
    * @param {number} count
    */
   setModuleJobsCompleted (moduleName, count) {
+    const filAddress = getFilAddress()
+    if (filAddress && moduleName in this.#perModuleJobStats) {
+      const diff = count - this.#perModuleJobStats[moduleName]
+      if (diff > 0) {
+        writeClient.writePoint(
+          new Point('jobs-completed')
+            .stringField('module', moduleName)
+            .stringField('wallet', createHash('sha256').update(filAddress).digest('hex'))
+            .stringField('station', getStationID())
+            .intField('value', diff)
```

Comment:
I find this a bit too much involved with details that are not relevant to reporting the number of jobs completed. I would expect that both wallet and station IDs will be added to all points; they are not specific to job stats. Are they?

Can we move the code setting `wallet` and `station` fields into our `telemetry` helper file? For example:


```js
exports.writePoint = function writePoint(point) {
  point          
    .stringField('wallet', createHash('sha256').update(filAddress).digest('hex'))
    .stringField('station', getStationID())
  writeClient.writePoint(point)
}
```

and example usage in `job-stats.js`:

```suggestion
        writePoint(
          new Point('jobs-completed')
            .stringField('module', moduleName)
            .intField('value', diff)
```

What do you think?

Reply:
This is a great idea! Done just like that